### DISCUSSION TITLE
Doesn't register inverter fault?

### DISCUSSION THREAD

**Discussion by MarkRS-UK** [nathanmarlor/foxess_modbus#647] [Support]
Marvellous HA extension, thanks.  It's recording all sorts of things looking quite right.

However, my Fox webpage is showing a 10 Minute Average Grid Voltage Fault at 14:49:05 (about two hours ago) but the modbus inverter_fault_code is showing "None" and claims its last change was three days ago.  I've just noticed that the log shows that the Inverter State changed at 14:50:02, to "Checking" and then a minute later to "On Grid".

Firstly I suppose, is this the right entity?  And after that, am I wrong in expecting to see something?

I have polling for updates enabled and set to every minute.
This is a FoxESS H1-5.0-E-AUX inverter.  
HA is Core 2024.7.2, Supervisor 2024.06.2, Operating System 12.4, Frontend 20240710.0

**Comment by maddin4711** [comment]
I can confirm this behavior. In my case, the error was displayed in the inverter, but not in Home Assistant.

H3-12.0-E inverter:
Master version: 1.98
Slave version: 1.03
Manager version: 1.78

HA Version 2024.7.1
Repository Version 1.12.1
![Error_not_in_HA](https://github.com/user-attachments/assets/8f48a2c5-e651-4908-ba2d-12bdc18acb1b)
